spec/solidus_starter_frontend_helper.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+require 'rails-controller-testing'
+require 'rspec/active_model/mocks'
+
+# Requires factories and other useful helpers defined in spree_core.
+require 'solidus_dev_support/rspec/feature_helper'
+require 'spree/testing_support/caching'
+require 'spree/testing_support/order_walkthrough'
+require 'spree/testing_support/translations' unless Spree.solidus_gem_version < Gem::Version.new('2.11')
+
+# Requires supporting ruby files with custom matchers and macros, etc,
+# in spec/support/ and its subdirectories.
+Dir["#{__dir__}/support/solidus_starter_frontend/**/*.rb"].sort.each { |f| require f }
+
+RSpec.configure do |config|
+  config.disable_monkey_patching!
+  config.infer_spec_type_from_file_location!
+  config.use_transactional_fixtures = false
+
+  # We currently have examples wherein we mock or stub method that do not exist on
+  # the real objects.
+  config.mock_with :rspec do |mocks|
+    mocks.verify_partial_doubles = false
```

Comment:
This would override a `mocks.verify_partial_doubles = true` config on the user's Rails app `spec_helper.rb` if one is set. We need to set this this to false in order to get the following tests to pass:

* `./spec/controllers/spree/base_controller_spec.rb`
* `./spec/controllers/spree/checkout_controller_spec.rb`
* `./spec/controllers/spree/products_controller_spec.rb`
* `./spec/requests/spree/orders_ability_spec.rb`

Reply:
Good to know. Yes, I also don't like this, what about trying to adjust specs that require the verification?

Reply:
@kennyadsl Hmm... Let me see if I can adjust the specs before I merge this PR. TODO.

Reply:
@kennyadsl Turns out that most of the stubs were no longer needed :)